src/Microsoft.Azure.WebJobs.Host/Executors/JobHostContextFactory.cs

```diff
@@ -126,7 +126,7 @@ public async Task<JobHostContext> Create(JobHost host, CancellationToken shutdow
                     executor.HostOutputMessage = hostOutputMessage;
                 }
 
-                IEnumerable<FunctionDescriptor> descriptors = functions.ReadAllDescriptors();
+                var descriptors = functions.ReadAllDescriptors();
```

Comment:
Any reason for this? Other than formatting, it's the only change in this file.

Reply:
Initially there were other changed in this file which I ended up scaling back. Will revert.